commit 62e50120f9721e7babaf0979e236e19ddd0fa029
Author: Anonymous <anony@example.com>
Date:   Sun May 25 04:42:24 2025 +0000

    norm from msb

diff --git a/src/core/include/lattice/hal/default/poly-impl.h b/src/core/include/lattice/hal/default/poly-impl.h
index 4da9ef54..3bc0cb0c 100644
--- a/src/core/include/lattice/hal/default/poly-impl.h
+++ b/src/core/include/lattice/hal/default/poly-impl.h
@@ -510,7 +510,7 @@ double PolyImpl<VecType>::Norm() const {
             maxVal = val > maxVal ? val : maxVal;
     }
     minVal = q - minVal;
-    return (minVal > maxVal ? minVal : maxVal).ConvertToDouble();
+    return (minVal > maxVal ? minVal : maxVal).GetMSB();
 }
 
 // Write vector x(current value of the PolyImpl object) as \sum\limits{ i = 0
